arangodb-net-standard/IndexApi/IndexApiClient.cs

```diff
@@ -144,36 +138,6 @@ public virtual async Task<IndexResponseBase> PostIndexAsync(IndexType indexType,
                 throw new System.ArgumentException("body is required", nameof(body));
             }
 
-            switch (indexType)
-            {
-                case IndexType.Persistent:
-                    {
-                        //uri += "#persistent";
-                        body.Type = "persistent";
-                    }
-                    break;
-                case IndexType.FullText:
-                    {
-                        //uri += "#fulltext";
-                        body.Type = "fulltext";
-                    }
-                    break;
-                case IndexType.TTL:
-                    {
-                        //uri += "#ttl";
-                        body.Type = "ttl";
-                    }
-                    break;
-                case IndexType.Geo:
-                    {
-                        //uri += "#geo";
-                        body.Type = "geo";
-                    }
-                    break;
-                default:
-                    throw new System.ArgumentException("Invalid index type");
```

Comment:
I removed this check thinking that if in future there is a new index type added in ArangoDB but someone hasn't updated the client yet, they can still submit the new index type. ArangoDB will return an error if the index type isn't supported, as confirmed by the new test `PostIndexAsync_ShouldThrow_WhenIndexTypeIsInvalid`.

Reply:
@DiscoPYF, I agree